### PR TITLE
docs: refactor hw_definitions.md

### PR DESCRIPTION
**Pull request by cleveHEX**

### CODE REVIEW

File: docs/configs/hw_definitions.md [hunk 1]
```diff
@@ -1,86 +1,42 @@
...
+## modbus_register_blocks
 
-Name of the file is the device code, which is then entered into the Evok configuration.
+Contains a list that defines the Modbus register groups. These registers must be placed consecutively. Each block is read in a separate request. All elements contain the following parameters:
 
-### modbus_register_blocks
+- `start_reg` - starting Modbus register address of the block
+- `count` - number of registers to be read
+- `frequency` - denominator of the scanning frequency set in [Evok configuration](./evok_configuration.md#modbustcp-modbusrtu) for this block
+- `type` - register type, defaults to `holding`, may be also `input`
```
Comment (kratochvil01) [inline docs/configs/hw_definitions.md:12]: Missing example. 

```
# This key defines which Modbus registers will be periodically read. Each block (also sometimes referred to as "group") is read once ever ["frequency"] read cycles
modbus_register_blocks:
    - start_reg   : 0
      count       : 10
      frequency   : 1

    - start_reg   : 500
      count       : 8
      frequency   : 10

    - start_reg   : 508
      count       : 8
      frequency   : 50
```

File: docs/configs/hw_definitions.md [hunk 2]
```diff
@@ -1,86 +1,42 @@
...
 
-- type - device type, supported devices are listed below.
-- count - number of devices of the type, register addresses incremented based on this number
+### DO (digital output)
 
-Other commands depend on the specific type of device.
+Allows you to add a digital output for the Modbus device.
 
-### Example
+- `val_coil` - coil register address
```
Comment (kratochvil01) [inline docs/configs/hw_definitions.md:27]: Most `*_coil` and `*_reg` parameters is only first register address. Real address is iterated with `count`. 
 Exception is for example `val_reg` in digital devices (DO, RO), but that will change when fix [this issue](https://github.com/UniPiTechnology/evok/issues/195).

**Review by kratochvil01 — CHANGES_REQUESTED** [review]
I would add to the parameter descriptions the data type that should be set there.

File: docs/configs/hw_definitions.md (hunk 1 above)
Comment (cleveHEX) [reply docs/configs/hw_definitions.md:12]: Example was migrated.

File: docs/configs/hw_definitions.md (hunk 2 above)
Comment (cleveHEX) [reply docs/configs/hw_definitions.md]: Added a `note`.

**Review by kratochvil01 — APPROVED** [review]

**Pull request merged**